src/config/2021.json

```diff
@@ -347,6 +376,16 @@
       "github": "hemanth",
       "twitter": "gnumanth"
     },
+    "fellowhuman1101": {
+      "name": "Jamie Indigo",
+      "teams": [
+        "authors"
+      ],
+      "avatar_url": "https://avatars.githubusercontent.com/u/52051775?v=4&s=200",
+      "website": "https://not-a-robot.com/",
+      "github": "fellowhuman1101",
+      "twitter": "Jammer_Volts"
+    },
```

Comment:
@fellowhuman1101 FYI adding your contributor info. Open a new PR to make any changes.